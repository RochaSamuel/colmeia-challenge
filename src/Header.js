import React from 'react'

const Header = () => {
    return (
        <nav>
            <div className="nav-wrapper grey lighten-5">
                <a href="/" className="brand-logo"><img id="logo" src="https://aulascolmeia.com.br/img/header-logo.png" alt='' /></a>
                <ul className="right">
                </ul>
            </div>
        </nav>
    );
}

export default Header