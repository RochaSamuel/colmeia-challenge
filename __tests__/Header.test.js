import React from 'react';
import { shallow } from 'enzyme';
import Header from '../src/Header';

describe('Testing Header Component', () => {
    it('should render correctly', () => {
        const wrapper = shallow(
            <Header />
        );

        expect(wrapper).toMatchSnapshot();
    });
});