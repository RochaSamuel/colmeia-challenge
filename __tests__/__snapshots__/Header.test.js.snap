// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Header Component should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="nav-wrapper grey lighten-5"
>
        <a
                className="brand-logo"
                href="/"
        >
                <img
                        alt="Logo colmeia"
                        id="logo"
                        src="https://aulascolmeia.com.br/img/header-logo.png"
                />
        </a>
        <ul
                className="right"
        />
</div>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <a
            className="brand-logo"
            href="/"
>
            <img
                        alt="Logo colmeia"
                        id="logo"
                        src="https://aulascolmeia.com.br/img/header-logo.png"
            />
</a>,
          <ul
            className="right"
/>,
        ],
        "className": "nav-wrapper grey lighten-5",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <img
              alt="Logo colmeia"
              id="logo"
              src="https://aulascolmeia.com.br/img/header-logo.png"
/>,
            "className": "brand-logo",
            "href": "/",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "Logo colmeia",
              "id": "logo",
              "src": "https://aulascolmeia.com.br/img/header-logo.png",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          "type": "a",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "right",
          },
          "ref": null,
          "rendered": null,
          "type": "ul",
        },
      ],
      "type": "div",
    },
    "type": "nav",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="nav-wrapper grey lighten-5"
>
          <a
                    className="brand-logo"
                    href="/"
          >
                    <img
                              alt="Logo colmeia"
                              id="logo"
                              src="https://aulascolmeia.com.br/img/header-logo.png"
                    />
          </a>
          <ul
                    className="right"
          />
</div>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <a
              className="brand-logo"
              href="/"
>
              <img
                            alt="Logo colmeia"
                            id="logo"
                            src="https://aulascolmeia.com.br/img/header-logo.png"
              />
</a>,
            <ul
              className="right"
/>,
          ],
          "className": "nav-wrapper grey lighten-5",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <img
                alt="Logo colmeia"
                id="logo"
                src="https://aulascolmeia.com.br/img/header-logo.png"
/>,
              "className": "brand-logo",
              "href": "/",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "Logo colmeia",
                "id": "logo",
                "src": "https://aulascolmeia.com.br/img/header-logo.png",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            "type": "a",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "right",
            },
            "ref": null,
            "rendered": null,
            "type": "ul",
          },
        ],
        "type": "div",
      },
      "type": "nav",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": false,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
}
`;
